OpenRA.Mods.Common/HitShapes/Capsule.cs

```diff
@@ -104,15 +104,17 @@ IEnumerable<IRenderable> IHitShape.RenderDebugOverlay(WorldRenderer wr, WPos ori
 			var offset2 = new WVec(aa.Y - bb.Y, bb.X - aa.X, 0);
 			offset2 = offset2 * Radius.Length / offset2.Length;
 
-			yield return new CircleAnnotationRenderable(a, Radius, 1, Color.Yellow);
-			yield return new CircleAnnotationRenderable(b, Radius, 1, Color.Yellow);
-			yield return new CircleAnnotationRenderable(aa, Radius, 1, Color.Yellow);
-			yield return new CircleAnnotationRenderable(bb, Radius, 1, Color.Yellow);
-			yield return new CircleAnnotationRenderable(origin, OuterRadius, 1,  Color.LimeGreen);
-			yield return new LineAnnotationRenderable(a - offset1, b - offset1, 1, Color.Yellow);
-			yield return new LineAnnotationRenderable(a + offset1, b + offset1, 1, Color.Yellow);
-			yield return new LineAnnotationRenderable(aa - offset2, bb - offset2, 1, Color.Yellow);
-			yield return new LineAnnotationRenderable(aa + offset2, bb + offset2, 1, Color.Yellow);
+			var shapeColor = disabled ? Color.LightGray : Color.Yellow;
```

Comment:
From your screenshot I find it relatively hard to distinguish between these two colours. Do you think it would be worth changing yellow to something else? Gold would offer slightly more contrast (probably not enough), or we could pick any other colour to make it even more clear.

Reply:
I see your point, but I'm not comfortable with using Gold over Yellow since that might bring the color a bit closer to GDI colors in TD and TS, potentially regressing contrast vs. the actor that the shape belongs to. And if I use a darker shade of gray for the disabled state, it becomes harder to see on non-snow terrain types.

I think modders will unfortunately need to have some good eyes no matter what we do here, so I'd rather leave it like this for now.